glyphs-reader/src/font.rs

```diff
@@ -975,6 +979,14 @@ impl RawFont {
         if let Some(kerning) = self.other_stuff.remove("kerning") {
             self.other_stuff.insert("kerningLTR".to_string(), kerning);
         };
+        for glyph in self.glyphs.iter_mut() {
+            if let Some(Plist::String(group)) = glyph.other_stuff.remove("leftKerningGroup") {
+                glyph.kern_left = Some(group.clone());
```

Comment:
I don't think this clone is necessary? `remove` should take ownership.

Reply:
Quite right, ty. Originally I had `.get`. I'm a bit surprised clippy didn't fault me for that one.